Upgrade Alt-1 to Rocky release

dougszumski opened the pull request.

markgoddard committed:
Synchronise configuration with Kayobe for Queens

Sync was performed with Kayobe @
ebf93b8dc187a9e4f40a5b1c0e0a58e13ab44006.

Change-Id: Ia269dc9e446594549ec652e2a952d3eb3b0dc07a

markgoddard committed:
Use master upper-constraints

Change-Id: I81f13085c6af09e01c8eacc04b9b9e1858f7affc

superpeng000 committed:
Change openstack-dev to openstack-discuss

Mailinglists have been updated. Openstack-discuss replaces openstack-dev.

Change-Id: I4d99d6d4e352f089423c9eddd32d1717828dbdab

markgoddard committed:
Synchronise with Kayobe

Pulls in configuration from Kayobe prior to cutting a stable/rocky
branch.

Change-Id: Id61bc95f88ddfca6ffc919bae7345704497e26c4

markgoddard committed:
Update .gitreview and tox.ini for stable/rocky

Change-Id: I00157cd78e5217d98830949156bead3fae798324

Commit:
OpenDev Migration Patch

This commit was bulk generated and pushed by the OpenDev sysadmins
as a part of the Git hosting and code review systems migration
detailed in these mailing list posts:

http://lists.openstack.org/pipermail/openstack-discuss/2019-March/003603.html
http://lists.openstack.org/pipermail/openstack-discuss/2019-April/004920.html

Attempts have been made to correct repository namespaces and
hostnames based on simple pattern matching, but it's possible some
were updated incorrectly or missed entirely. Please reach out to us
via the contact information listed at https://opendev.org/ with any
questions you may have.

dougszumski committed:
Upgrade Monasca to Rocky release

This adds the following passwords to passwords.yml:

- monasca_grafana_admin_password
- monasca_agent_password

markgoddard commented on etc/kayobe/kolla.yml:
```diff
@@ -37,7 +37,7 @@ kolla_ansible_source_url: https://github.com/stackhpc/kolla-ansible.git
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-kolla_ansible_source_version: refs/tags/stackhpc/6.1.0.6
+kolla_ansible_source_version: stackhpc/rocky
```
Can't really merge with this in place until we upgrade the rest.

dougszumski replied:
Yeah, i'm just working off this branch for now and will use the relevant tags for the upgrade.

markgoddard committed:
Update remaining components to Rocky

Kolla: stackhpc/7.0.2.1
Kolla-Ansible: stackhpc/7.1.0.1
NGS: stackhpc/1.2.0.1
Magnum: stackhpc/7.1.0.1
Networking-MLNX: Use upstream
Sahara: Stay on Pike release stackhpc/7.0.2.1

markgoddard committed:
Merge upstream kayobe-config stable/rocky

dougszumski commented on etc/kayobe/kolla/globals.yml:
```diff
@@ -19,29 +19,41 @@ monasca_default_authorized_roles:
+influxdb_tag: 7.1.0.1
+ironic_tag: 7.0.2.1-1
+iscsid_tag: 7.0.2.1-1
+kafka_tag: 7.1.0.1
```
Now I noticed I named the tag incorrectly - thanks for the tool.

dougszumski commented:
+1 from me, but I can't approve your changes since I created the PR.

markgoddard committed:
Disable chrony container

Use ntpd on the host instead.

markgoddard committed:
Update passwords.yml

Adds passwords for nsxv3 and onos.

markgoddard committed:
Bump kolla-ansible to stackhpc/7.1.0.2

Adds monasca-log-transformer fix for logstash_patterns.

markgoddard committed:
Install networking-mlnx in neutron-server

While we no longer need to use a fork of networking-mlnx, we do still need to
install it in the neutron-server image.

markgoddard committed:
Use a fork of nova, stackhpc/18.2.0.1

Fixes nova instance tear down, see
https://bugs.launchpad.net/nova/+bug/1815799.

markgoddard reviewed (approved):
Upgrade went well overall, now seems to be working with these changes.
